connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -122,6 +123,14 @@ export async function syncZendeskBrandActivity({
     );
   }
 
+  // deleting the data not allowed anymore
```

Comment:
I'm not sure I understand the comment with the following code 🙈

Reply:
changed the comment into `deleting the tickets/help center if not allowed anymore`